pkg/psalabelsyncer/podsecurity_label_sync_controller.go

```diff
@@ -372,6 +372,13 @@ func (c *PodSecurityAdmissionLabelSynchronizationController) syncNamespace(ctx c
 		}
 	}
 
+	if ns.Annotations[securityv1.MinimallySufficientPodSecurityStandard] != string(psaLevel) {
+		nsApplyConfig = nsApplyConfig.WithAnnotations(map[string]string{
+			securityv1.MinimallySufficientPodSecurityStandard: string(psaLevel),
+		})
```

Comment:
Nit: Do we need the `nsApplyConfig =` here? Everywhere else values get updated on the apply configuration it looks like its done without it.